<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>600</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="keyboard">
    <widget class="QLabel" name="background_key">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>600</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>graphic/emoto2.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_key">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>30</y>
       <width>534</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-color: rgb(146, 150, 150);
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)
   </string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="button_key1">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>130</y>
       <width>150</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri Light</family>
       <pointsize>36</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_key2">
     <property name="geometry">
      <rect>
       <x>259</x>
       <y>130</y>
       <width>150</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_key3">
     <property name="geometry">
      <rect>
       <x>437</x>
       <y>130</y>
       <width>150</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_key6">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>250</y>
       <width>150</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_key7">
     <property name="geometry">
      <rect>
       <x>259</x>
       <y>250</y>
       <width>150</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_key4">
     <property name="geometry">
      <rect>
       <x>615</x>
       <y>130</y>
       <width>150</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_key5">
     <property name="geometry">
      <rect>
       <x>793</x>
       <y>130</y>
       <width>150</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_key0">
     <property name="geometry">
      <rect>
       <x>793</x>
       <y>250</y>
       <width>150</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_key9">
     <property name="geometry">
      <rect>
       <x>615</x>
       <y>250</y>
       <width>150</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_key8">
     <property name="geometry">
      <rect>
       <x>437</x>
       <y>250</y>
       <width>150</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_keyCancel">
     <property name="geometry">
      <rect>
       <x>763</x>
       <y>380</y>
       <width>180</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 24pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>CANCEL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_keyBack">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>380</y>
       <width>180</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 24pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_keySet">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>500</y>
       <width>341</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 24pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>SET</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="screensaver">
    <widget class="QLabel" name="background_screen">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>600</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>graphic/emoto2.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="button_screen">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>644</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-width: 1px;
	border-radius: 1px;
	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_screen">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>20</y>
       <width>854</width>
       <height>480</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <zorder>background_screen</zorder>
    <zorder>label_screen</zorder>
    <zorder>button_screen</zorder>
   </widget>
   <widget class="QWidget" name="access">
    <widget class="QLabel" name="label_start">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>600</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>graphic/emoto1.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="button_start">
     <property name="geometry">
      <rect>
       <x>187</x>
       <y>350</y>
       <width>650</width>
       <height>120</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri Light</family>
       <pointsize>36</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_date_start">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>10</y>
       <width>411</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>X, 00,00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_hour_start">
     <property name="geometry">
      <rect>
       <x>788</x>
       <y>10</y>
       <width>231</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="access_granted">
    <widget class="QLabel" name="background_access">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>600</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>graphic/emoto2.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="button_access">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>600</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri Light</family>
       <pointsize>36</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="settings">
    <widget class="QLabel" name="background_key_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>600</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>graphic/emoto2.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_set">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>30</y>
       <width>634</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-color: rgb(146, 150, 150);
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)
   </string>
     </property>
     <property name="text">
      <string>WHAT DO YOU WANT TO DO?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="button_key1_2">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>130</y>
       <width>500</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri Light</family>
       <pointsize>36</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>ENROLL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_key6_2">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>230</y>
       <width>500</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>CHECK LOGS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_key2_2">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>330</y>
       <width>500</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 36pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>VERIFY???</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_setBack">
     <property name="geometry">
      <rect>
       <x>763</x>
       <y>380</y>
       <width>180</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	border-style: solid;
	border-width: 3px;
	border-radius: 16px;
	color: rgb(146, 150, 150);
	font: 300 24pt &quot;Calibri Light&quot;;
	background-image (&quot;C:\Users\mjszo\Desktop\1\graphic\blang.png&quot;)</string>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
